tests/gocase/unit/introspection/introspection_test.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
May I ask the meaning of the filename? I think something like `ping_test.go` will be more simple.

Reply:
I did this to imitate [redis](https://github.com/redis/redis/blob/unstable/tests/unit/introspection.tcl#L3-L5). Kvrocks also has introspection.tcl

Reply:
Oh, got it. Thanks.